Add state management quickstart (#2320)
dapr/docs: daprdocs/content/en/getting-started/quickstarts/_index.md

| Quickstarts | Description |
| ----------- | ----------- |

> | [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) | Invoke a method using HTTP proxying with Dapr's Service Invocation building block. |

Reviewer: ```suggestion
| [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) |  Call a method on another service using the service invocation API. |
```

| ----------- | ----------- |
| [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) | Invoke a method using HTTP proxying with Dapr's Service Invocation building block. |

> | [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Get started with Dapr's Publish and Subscribe building block. |

Reviewer: ```suggestion
| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Send and receive messages using the publish and subscribe API. |
```

> | [State Management]({{< ref statemanagement-quickstart.md >}}) | Create a microservice to demonstrate Dapr's state management API. |

Reviewer: ```suggestion
| [State Management]({{< ref statemanagement-quickstart.md >}}) | Create stateful applications using the state management API. |
```
